src/axe-raw-sarif-converter.test.ts

```diff
@@ -71,27 +72,65 @@ describe('AxeRawSarifConverter', () => {
     });
 
     describe('convert', () => {
-        it('outputs a sarif log whose run uses the invocationsProvider to populate the invocations property', () => {
-            const stubEnvironmentData = {
+        let stubEnvironmentData: EnvironmentData;
+        let stubToolProperties: Sarif.Run['tool'];
+        let stubInvocations: Sarif.Invocation[];
+
+        let axeToolPropertyProviderMock: IMock<() => Sarif.Run['tool']>;
+        let invocationProviderMock: IMock<
+            (environmentData: EnvironmentData) => Sarif.Invocation[]
+        >;
+
+        beforeEach(() => {
+            stubEnvironmentData = {
                 targetPageUrl: 'stub_url',
             } as EnvironmentData;
+            stubToolProperties = {
+                name: 'stub_tool_property',
+            };
+            stubInvocations = [{ commandLine: 'stub_invocation' }];
 
-            const stubInvocations: Sarif.Invocation[] = [
-                { commandLine: 'stub_invocation' },
-            ];
+            axeToolPropertyProviderMock = Mock.ofInstance(getAxeToolProperties);
+            invocationProviderMock = Mock.ofInstance(getInvocations);
 
-            const invocationProviderMock: IMock<
-                (environmentData: EnvironmentData) => Sarif.Invocation[]
-            > = Mock.ofInstance(getInvocations);
+            axeToolPropertyProviderMock
+                .setup(ap => ap())
+                .returns(() => stubToolProperties)
+                .verifiable(Times.once());
 
             invocationProviderMock
                 .setup(ip =>
                     ip(It.isObjectWith<EnvironmentData>(stubEnvironmentData)),
                 )
                 .returns(() => stubInvocations)
                 .verifiable(Times.once());
+        });
+
+        it('outputs a sarif log whose run uses the axeToolPropertyProvider to populate the tool property', () => {
+            const testSubject = new AxeRawSarifConverter(
+                axeToolPropertyProviderMock.object,
+                invocationProviderMock.object,
```

Comment:
>invocationProviderMock.object [](start = 16, length = 29)

I think it would be more clear to use a stub instead of a verifiable mock for the cases you aren't verifying the calls for (ie, for the invocation provider in the axeToolProperty test and for the axeToolPropertyProvider in the invocation provider test); using a verifiable mock implies that it's probably a test bug to not be verifying it, imo.